### PR TITLE
Release 1.96.0

### PR DESCRIPTION
**Pull request by derekblank**
Release for Gutenberg Mobile 1.96.0

## Related PRs

- Gutenberg: https://github.com/WordPress/gutenberg/pull/50935
- WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/18508/
- WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/20740

## Extra PRs that Landed After the Release Was Cut

No extra PRs yet. 🎉

### Add disabled style to Cell component
- **PR:** https://github.com/WordPress/gutenberg/pull/50665
- **Issue:** N/A

### Fix undo/redo history when inserting a link configured to open in a new tab
- **PR:** https://github.com/WordPress/gutenberg/pull/50460
- **Issue 1:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2091
- **Issue 2:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/1160
- **Issue 3:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/1161

### Disable details settings for not belonged VideoPress videos
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5782
- **Issue:** https://github.com/Automattic/jetpack/pull/30759

### Enable VideoPress block on Android
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5780
- **Issue:** N/A

### Tapping any type of nested block moves focus to the nested block directly, rather than requiring multiple taps to navigate down each nesting levels
- **PR:** https://github.com/WordPress/gutenberg/pull/50672
- **Issue:** N/A

### List block - Fix merging nested list into paragraph
- **PR:** https://github.com/WordPress/gutenberg/pull/50701
- **Issue:** https://github.com/WordPress/gutenberg/pull/50634

## Test plan

Once the installable builds of the main apps are ready, perform a quick smoke test of the editor on both iOS and Android to verify it launches without crashing. We will perform additional testing after the main apps cut their releases.

## Release Submission Checklist

- [ ] Verify Items from test plan have been completed
- [x] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [x] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [x] Bundle package of the release is updated.

### CODE REVIEW

**Comment by mokagio** [comment]
@derekblank apologies for [the `Podfile.lock` inconvenience](https://buildkite.com/automattic/gutenberg-mobile/builds/5855#0188507c-290a-44d8-ad94-5d5e003d673a). 

image

I fixed the issue in https://github.com/wordpress-mobile/gutenberg-mobile/commits/1815c5fbcafd5a5c39f9d78b37530a2d755c17ab and updated the release automation so that this should not occur again, https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/116

**Comment by fluiddot** [comment]
> Verify Items from test plan have been completed

I've unmarked this checkbox as the test plan hasn't been completed yet. Next Tuesday, we'll perform it.

> Bundle package of the release is updated.

I checked the commits and confirmed that the bundle was updated, hence I'm marking this checkbox.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+
+
+1.96.0
+---
 * [*] Add disabled style to `Cell` component [https://github.com/WordPress/gutenberg/pull/50665]
+* [**] Fix undo/redo history when inserting a link configured to open in a new tab [https://github.com/WordPress/gutenberg/pull/50460]
 * [**] Disable details settings for not belonged VideoPress videos [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5782]
-* [***] Enable VideoPress block on Android (only available in Simple WPCOM sites) [#5780]
+* [***] Enable VideoPress block on Android (only available in Simple WPCOM sites) [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5780]
 * [**] Tapping any type of nested block moves focus to the nested block directly, rather than requiring multiple taps to navigate down each nesting levels. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5781]
```
Comment (geriux) [inline RELEASE-NOTES.txt:11]: Hey there 👋 I just noticed this change is not being included in the main apps release notes, is there a reason for that? It's not a major change but thought I'd ask.

Comment (fluiddot) [reply RELEASE-NOTES.txt:11]: Ah, good catch. I overlooked this entry when reviewing the PRs, sorry for that. That said, I think we should incorporate it, maybe it's not too late to incorporate it into the release notes 🤞 .

Comment (fluiddot) [reply RELEASE-NOTES.txt:11]: I've created a couple of PRs to update the release notes into version `22.5`:
- [WP-Android](https://github.com/wordpress-mobile/WordPress-Android/pull/18551)
- [WP-iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/20767)

Comment (geriux) [reply RELEASE-NOTES.txt:11]: Approved! Thank you for adding it and wrangling the PRs! 🙇

**Comment by SiobhyB** [comment]
## 🟢 Writing Flow checklist (iOS)

**Build:** 22.5.0.0
**Device:** iPhone XR (iOS 16.5)

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/multiline-components.md#tc001) - Multiline components
  - [x] Quote block
  - [x] Verse block
  - [x] Preformatted block
  - [ ] Code block (DEV only)
  - [x] Pullquote block
#### Rich Text Format
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc001) - Bold, Italic, strikethrough buttons
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc001) - Highlight selected text
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc001) - Highlight text without selection
   - ⚠️ Encountered https://github.com/WordPress/gutenberg/issues/42714#issuecomment-1568430244
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc002) - Alignment buttons
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc003) - Alignment Split
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc004) - Link button works without selection
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc005) - Link button works with a selected word
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc006) - Adding a link from a copied URL
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc008) - Test formatting doesn't remove leading or trailing whitespace
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc006) - Splitting/merge list block
#### Undo / Redo - Test Cases
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc001) - Undo/redo block actions
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc002) - Undo/redo text
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc003) - Undo/redo text format

**Comment by SiobhyB** [comment]
## 🟢 Unsupported Block Tests (iOS)

**Build:** 22.5.0.0
**Device:** iPhone XR (iOS 16.5)

- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by SiobhyB** [comment]
## 🟢 Included Changes (iOS)

**Build:** 22.5.0.0
**Device:** iPhone XR (iOS 16.5)

- [x] https://github.com/WordPress/gutenberg/pull/50665
- [x] https://github.com/WordPress/gutenberg/pull/50460
- [x] https://github.com/wordpress-mobile/gutenberg-mobile/pull/5782
- [x] ~https://github.com/wordpress-mobile/gutenberg-mobile/pull/5780~
- [x] https://github.com/WordPress/gutenberg/pull/50672
- [x] https://github.com/WordPress/gutenberg/pull/50701

**Comment by SiobhyB** [comment]
## 🟢 Test Suite 2 (iOS)

**Build:** 22.5.0.0
**Device:** iPhone XR (iOS 16.5)

### DarkMode - 1

- [x] Gallery block - Dark mode - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/darkmode.md#tc001)
- [x] Shortcode block - Dark mode - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/darkmode.md#tc002)
- [x] Media Text block - Dark mode - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/darkmode.md#tc003)

### Group - 1

- [x] Group - Deep nesting is possible (verify iOS does not crash) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc001)
- [x] Group - Check if Group placeholder is visible for the unselected state - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc002)
- [x] Group - Check if Group placeholder is render in nested structure - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc003)
- [x] Group - Nested block have proper border styling - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc004)
- [x] Group - Breadcrumbs on FloatingToolbar is properly displayed - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc005)

### Buttons-1

- [x] Buttons block - Button's wrapper grows properly - [TC019](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc019)
- [x] Buttons block - Removing exactly one Button (when Buttons contain more of them) - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc002)
- [x] Buttons block - Wrapping Buttons - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc003)

### Button-2

- [x] Buttons block - Buttons alignment options - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc004)
- [x] Buttons block - Settings: Open in new tab - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc007)
- [x] Buttons block - Settings: Link rel - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc008)

### Button-3

- [x] Buttons block - Settings: Link URL - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc006)
- [x] Buttons block - Settings: Remove link - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc010)
- [x] Buttons block - Settings: Synchronize with button options - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc009)

### Color Settings - 1
- [x] Colors Segmented Controls - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc001)
- [x] Scroll color palette to start - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc002)
- [x] Scroll color palette to the end - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc003)
   - ⚠️ Encountered https://github.com/WordPress/gutenberg/issues/30878.

### Image - 1
- [x] Image block - Insert image from device (failing) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc001)
- [x] Image block - Insert image from device (cancel) - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc002)
- [x] Image block - Add Caption - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc003)
- [x] Image block - Close/Re-open post with an ongoing image upload - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc004)
- [x] Image block - Close post with an ongoing image upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/image.md#tc005)

### Drag and Drop Blocks - 1
- [x] Drag and Drop Blocks - Drag block to the top of the content - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/drag-and-drop-blocks.md#tc001)
- [x] Drag and Drop Blocks - Drag block to the bottom of the content - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/drag-and-drop-blocks.md#tc002)

### VideoPress - 2
- [x] VideoPress block - Player - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc003)
- [x] VideoPress block - Private video - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc005)
- [x] VideoPress block - Upload limit for Free users - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc006)

**Comment by geriux** [comment]
## 🟢 Writing Flow checklist (Android)
**Build:** [22.5-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/22.5-rc-1)
**Device:** Samsung A23 5G (Android 13)

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/multiline-components.md#tc001) - Multiline components
  - [x] Quote block
  - [x] Verse block
  - [x] Preformatted block
  - [ ] Code block (DEV only)
  - [x] Pullquote block
#### Rich Text Format
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc001) - Bold, Italic, strikethrough buttons
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc001) - Highlight selected text
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc001) - Highlight text without selection
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc002) - Alignment buttons
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc003) - Alignment Split
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc004) - Link button works without selection
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc005) - Link button works with a selected word
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc006) - Adding a link from a copied URL
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc008) - Test formatting doesn't remove leading or trailing whitespace
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc006) - Splitting/merge list block
#### Undo / Redo - Test Cases
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc001) - Undo/redo block actions
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc002) - Undo/redo text
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc003) - Undo/redo text format

**Comment by geriux** [comment]
## ⚪️ Unsupported Block Tests (Android)
**Build:** [22.5-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/22.5-rc-1)
**Device:** Samsung A23 5G (Android 13)

- [ ] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [ ] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [ ] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [ ] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [ ] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [ ] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by geriux** [comment]
## 🟢 New changes (Android)
**Build:** [22.5-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/22.5-rc-1)
**Device:** Samsung A23 5G (Android 13)

- [x] https://github.com/WordPress/gutenberg/pull/50665
- [x] https://github.com/WordPress/gutenberg/pull/50460
- [x] https://github.com/wordpress-mobile/gutenberg-mobile/pull/5782
- [x] https://github.com/wordpress-mobile/gutenberg-mobile/pull/5780
- [x] https://github.com/WordPress/gutenberg/pull/50672
- [x] https://github.com/WordPress/gutenberg/pull/50701

**Comment by geriux** [comment]
## ⚪️ Test Suite 1 (Android)
**Build:** [22.5-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/22.5-rc-1)
**Device:** Samsung Galaxy S20 FE 5G (Android 11)

### Gallery - 1

- [x] Gallery block - Close/Re-open post with an ongoing image upload - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc001)
- [x] Gallery block - Close post with an ongoing image upload - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc002)

### Gallery - 2

- [x] Gallery block - Insert image from device (failing) - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc006)
- [x] Gallery block - Insert image from device (cancel) - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc009)
- [x] Gallery block - Try adding same images from WP Media library and moving the images around - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc010)

### MediaText - 1

- [ ] Media Text block - Insert image from device (failing) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc001-i)
  - 🔴 CRASH: Fixed by https://github.com/wordpress-mobile/WordPress-Android/pull/18544
- [x] Media Text block - Close/Re-open post with an ongoing image upload - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc003-i)

### MediaText - 2

- [ ] Media Text block - Insert video from device - [TC002-v](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc002-v)
  - 🔴 CRASH: Fixed by https://github.com/wordpress-mobile/WordPress-Android/pull/18544
- [ ] Media Text block - Insert image from device - [TC002-i](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-text.md#tc002-i)
  - 🔴 CRASH: Fixed by https://github.com/wordpress-mobile/WordPress-Android/pull/18544

### MultipleUpload - 1

- [x] Simultaneous uploads - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/media-interaction.md#tc001)

### Cover - 1

- [x] Cover - Add image background from WordPress Media library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc001)
- [x] Cover - Image focal point is rendered properly - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc002)
- [x] Cover - Custom color is rendered properly - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/cover.md#tc003)

### Editor Theme - 1

- [x] Default Colors - Check that default colors still load - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc001)
- [x] Default Gradients - Check that default gradients still load - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc002)
- [x] Custom Colors - Check that custom colors load in the editor - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc003)

### Social Icons - 1
- [x] Active icon gets product color - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/social-icons.md#tc001)
- [x] Social icon forwarding to the link - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/social-icons.md#tc002)

**Pull request merged**